### PR TITLE
Reinstate DSL File Transformer Factory

### PR DESCRIPTION
**Pull request by garyrussell**

### CODE REVIEW

File: spring-integration-file/src/main/java/org/springframework/integration/file/dsl/FileTransformers.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ * @since 5.2
+ *
+ */
+public final class FileTransformers {
+
+	private FileTransformers() {
+		super();
+	}
+
+	public static FileToStringTransformer fileToString() {
```
Comment (artembilan) [inline spring-integration-file/src/main/java/org/springframework/integration/file/dsl/FileTransformers.java]: Can't we have these simple factory methods just directly in the `Files` class?
And make that `final`, too 😉

**Review by artembilan — CHANGES_REQUESTED** [review]

File: spring-integration-file/src/main/java/org/springframework/integration/file/dsl/Files.java [hunk 2]
```diff
@@ -226,4 +233,37 @@ public static FileToByteArrayTransformer toByteArrayTransformer(boolean deleteFi
 		return fileToByteArrayTransformer;
 	}
 
+	/**
+	 * Create a {@link FileToStringTransformer} with {@link Charset#defaultCharset()}.
+	 * @return the transformer.
+	 * @since 5.2
+	 */
+	public static FileToStringTransformer fileToString() {
```
Comment (artembilan) [inline spring-integration-file/src/main/java/org/springframework/integration/file/dsl/Files.java]: Wait (and sorry), looks like we have there already here:
```
public static FileToStringTransformer toStringTransformer(String charset) {
...
public static FileToByteArrayTransformer toByteArrayTransformer() {
```

😢

**Review by artembilan — CHANGES_REQUESTED** [review]
Time to close your PR altogether? 😉

File: spring-integration-file/src/main/java/org/springframework/integration/file/dsl/Files.java (hunk 2 above)
Comment (garyrussell) [reply spring-integration-file/src/main/java/org/springframework/integration/file/dsl/Files.java]: Doh

**Pull request closed without merging**